plugins/chat/assets/stylesheets/common/chat-message-images.scss

```
@@ -21,9 +21,23 @@ $max_image_height: 150px;
   .chat-message-collapser
     .chat-message-collapser-header
     + div
-    .chat-message-collapser-youtube {
+    .chat-message-collapser-lazy-video {
     object-fit: contain;
     height: $max_image_height;
     width: calc(#{$max_image_height} / 9 * 16);
   }
+
+  // Prevent overflow of lazy-yt images
```

```suggestion
  // Prevent overflow of old lazy-yt images
  // TODO: remove in December 2023
```

Since we're rebaking everything, it makes sense to schedule a cleanup here in 6+ months, does that make sense?